src/interpreter/mod.rs

```diff
@@ -31,6 +32,10 @@ pub struct EvalContext<'a, 'tcx: 'a> {
     /// The results of the type checker, from rustc.
     tcx: TyCtxt<'a, 'tcx, 'tcx>,
 
+    /// The Session, from rustc.
+    /// Used to extract info from other crates
+    session: &'a Session,
```

Comment:
darn it ^^ I thought it was somewhere there, but couldn't find it. Rustdocs are overwhelmed with TyCtxt